src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,13 @@
-export const TodoList = () => {};
```

Comment:
There are two exports with the same name `TodoList`. Consider removing the first export statement. 🤔